lively.morphic/svg-morph/svg-morph.js

```diff
@@ -140,15 +152,107 @@ export class SVGMorph extends Morph {
     selection_node.attr({
       'fill-opacity': 0.0,
       stroke: '#000',
-      'stroke-width': 2,
-      'pointer-events': 'none'
+      'stroke-width': 1,
+      'pointer-events': 'none',
+      'stroke-dasharray': '5, 3'
     });
     tar.after(selection_node);
     selection_node.front();
   }
 
   get isSVGMorph () { return true; }
 
+  getControlPoints () {
+    this.removeAllControlPoints();
+
+    const tar = SVG(this.target);
+    const targetPath = SVG(this.target).array();
+
+    for (let i = 0; i < targetPath.length; i++) {
+      let element = targetPath[i];
+      switch (element[0]) {
+        case 'Z':
+          break;
+        default:
+          let defaultPoint = this.createControlPointAt(i, element[element.length - 2], element[element.length - 1], 'yellow');
+          tar.after(defaultPoint);
+          defaultPoint.front();
+      }
+      let lastElement = element;
+    }
+  }
+
+  createControlPointAt (id, x, y, color) {
+    const t = SVG(this.svgPath);
+    let point = t.circle();
+
+    point.attr({
```

Comment:
Maybe we should think of a dict with default settings like the color. Not needed in this PR just an idea

Reply:
I think that is a good idea...
since I started creating points with different colors for different line commands (as stated above) it is just some "old code" which can easily be changed